service/models.py

```diff
@@ -50,12 +48,18 @@ class Recommendation(db.Model):
     """
     logger = logging.getLogger('flask.app')
     app = None
+	
+	
     id = db.Column(db.Integer, primary_key=True)
     product_id = db.Column(db.Integer)
     customer_id = db.Column(db.Integer)
     recommend_type = db.Column(db.String(63))
     recommend_product_id = db.Column(db.Integer)
+	
 
+    def __repr__(self):
+        return '<Recommendation %r>' % (self.product_id)
```

Comment:
The tests won't cover this line, can you please wrote a test for it or delete it? I don't know why we need it here.